kaldb/src/test/java/com/slack/kaldb/schema/SchemaConfigTest.java

```diff
@@ -0,0 +1,97 @@
+package com.slack.kaldb.schema;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import com.slack.kaldb.metadata.schema.SchemaUtil;
+import com.slack.kaldb.proto.schema.Schema;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class SchemaConfigTest {
+
+  @Test
+  public void testParsingEmptySchema() {
```

Comment:
We should add one for a malformed schema as well, and one that points to a file that doesn't exist.